packages/cli/test/scripts/create.test.js

```diff
@@ -117,15 +117,10 @@ contract('create script', function([_, owner]) {
     });
 
     it('should initialize a proxy using scientific notation', async function() {
-      const proxy = await createProxy({ contractAlias, network, txParams, initMethod: 'initialize', initArgs: ["20e10"], networkFile: this.networkFile });
+      const proxy = await createProxy({ contractAlias, network, txParams, initMethod: 'initialize', initArgs: [Number("20e10")], networkFile: this.networkFile });
       (await proxy.value()).toString().should.be.eq('200000000000');
     });
 
-    it('should initialize a proxy using large number on scientific notation', async function() {
```

Comment:
This test was removed because ethers/abi-coder checks that the incoming number is within uint256 bounds.